createWebNavigator.js

```diff
@@ -44,29 +43,19 @@ function CustomStackNavigator(props) {
                         )
                     }
 
-                    if (route.name !== 'Chat' && route.name !== 'LeftHandNav' && i !== lastRHPIndex) {
-                        return null;
-                    }
-
-                    if (!isRHPOnTopOfStack) {
-                        return null;
-                    }
-
-                    return (
-                        <View
-                            key={route.key}
-                            style={{width: '100%', height: '100%', position: 'absolute'}}
-                        >
-                            <View style={{flexDirection: 'row', width: '100%', height: '100%'}}>
-                                <Pressable style={{flex: 1}} onPress={() => navigation.goBack()}>
-                                    <View style={{backgroundColor: 'rgba(0, 0, 0, 0.4)', flex: 1, width: '100%', height: '100%'}} />
-                                </Pressable>
-                                <View style={{width: 375}}>
-                                    {descriptors[route.key].render()}
-                                </View>
+                    if (isRHPOnTopOfStack && i === state.routes.length - 1) {
+                        return (
+                            <View
+                                key={route.key}
+                                style={{width: '100%', height: '100%', position: 'absolute'}}
+                            >
+                                <RHPContainer navigation={navigation} contentStyle={{width: 375}}>
+                                        {descriptors[route.key].render()}
+                                </RHPContainer>
                             </View>
-                        </View>
-                    );
+                        );
+                    }
+                    return null;
```

Comment:
I'm not convinced we should do it at all yet. But if we end up wanting to use the web navigator for small screens I think we should clean up the logic so it's split into two separate functions that return the correct layout e.g.

```
<LargeFormatView />
<SmallFormatView />
```